framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -707,10 +707,13 @@ public AnnotationMirror getAnnotationFromJavaExpressionString(
 
         AnnotationMirror annotationMirror = null;
         if (CFAbstractStore.canInsertReceiver(expressionObj)) {
-            Value value = getStoreBefore(tree).getValue(expressionObj);
-            if (value != null) {
-                annotationMirror =
-                        AnnotationUtils.getAnnotationByClass(value.getAnnotations(), clazz);
+            Store store = getStoreBefore(tree);
```

Comment:
Actually this should just be reverted; I didn't mean to include it in the PR (and thought I'd removed it). Sorry about that.